core/src/main/resources/lib/form/select/select.js

```diff
@@ -43,6 +43,10 @@ function updateListBox(listBox,url,config) {
 Behaviour.specify("SELECT.select", 'select', 1000, function(e) {
 
         function hasChanged(selectEl, originalValue) {
+            if (selectEl.options.length == 0) {
+                // There are no options to select, so there is no possible change
```

Comment:
At any rate, if it worked before it will work now, and if it was `selectEl.options.length == 0` before a JS error happened so this PR is not really changing any previous behavior, just preventing a JS error.